tests/test_lion8b.py

```diff
@@ -498,6 +498,7 @@ def _set_state_dict_type(model: nn.Module):
             torch.testing.assert_close(d_orig['errors'], d_new['errors'])
 
 
+@pytest.mark.skip('Avoiding rare flakiness; be sure to run manually!')
```

Comment:
can you jira for adding retries to this test/making it not flaky? can put in the foundry backlog epic

Reply:
I can just fix it right now if you have ideas about how to do that. Do you think a few retries would be sufficient? I was just kinda erring on the side of caution since I can't guarantee execution time.

Reply:
since we hadn't noticed it for a while i had assumed that a few retries would be sufficient. im fine if you just add some retries and then we revisit if it is still flaky.

Reply:
added retries + relaxed the atol when checking timing